src/shared/components/case-viewer/services/case.resolver.spec.ts

```diff
@@ -222,6 +225,25 @@ describe('CaseResolver', () => {
       expect(navigationNotifierService.announceNavigation).toHaveBeenCalledWith({action: NavigationOrigin.NO_READ_ACCESS_REDIRECTION});
 
     });
+
+    it('should redirect to no results found page when case reference search from header is not found', () => {
+      const error = {
+        status: 400
+      };
+      casesService.getCaseViewV2.and.returnValue(Observable.throw(error));
```

Comment:
@johnbenjamin-hmcts - `Observable.throw(error)` is deprecated; you can use `throwError(error)` from RxJS 6 onwards.

Reply:
Thank you @Daniel-Lam, I have fixed this now.